### ISSUE TITLE
[Feature Request] GPS data in coordinate measurement

### ISSUE THREAD

**Issue by yawstick** [OpenDroneMap/WebODM#537]
Modify the format of the GPS lat/lon data displayed when doing a coordinate measurement on a 3d map. The current format is not so user friendly.

**Comment by smathermather** [comment]
Ah, the git issue bot strikes again.

**Comment by smathermather** [comment]
I think you can prevent this by putting “Feature request” in brackets.

Anyway, based on the community dialog, the current coordinates are in UTM with an offset. At the very least, having these display in UTM, and better yet in geographic coordinates is what you’re looking for, correct?

**Comment by yawstick** [comment]
yes that is correct... preferably decimal degrees lat/lon... it would then match what you see when your mouse is hovering over the 2d map.

My first time here...I will remember the bracket thing

**Comment by smathermather** [comment]
The high-level would be to convert the coordinates with offsets to their UTM equivalent and then transform those to geographic coordinates, and display the converted version.

**Comment by pierotofy** [comment]
Yup. Part of the difficulty is that UTM coordinates without offsets are large numbers, and WebGL uses floats to represent vertex coordinates, so we can't just add the offsets to the vertices and call it done. I like how glTF's Cesium RTC extension handles it. https://github.com/KhronosGroup/glTF/tree/master/extensions/1.0/Vendor/CESIUM_RTC

Perhaps just recording what the UTM offset is and then adding it for visualization/measurement purposes would be a reasonable solution.

**Comment by smathermather** [comment]
Yes, the simplest is to just display the correct UTM values, but something like proj4js could convert to geographic pretty easily too, I think.
